### ISSUE TITLE
Python 3.4 processing of content results in CRITICAL: unorderable types: Article() < Article()

### ISSUE THREAD

**Issue by salvor7** [getpelican/pelican-themes#483]
It doesn't look like this issue made it over here from the pelican repo. 
https://github.com/getpelican/pelican/issues/1591

I just experience this error with zurb-F5-basic myself. 

Sorry if this was raised under another name that I'm not seeing for what it is. My searches of the repo and closed tickets didn't turn up anything like it.

**Comment by joshmorel** [comment]
Hi salvor7,

Seems like this "just worked" with Python 2 but with Python 3 is throwing an error. A simple fix, but this affecting other themes and I would like to fix all at the same time  - I could do this perhaps tomorrow but would need the PR to be accepted, etc. 

In the interim, if you want this to work, in  zurb-f5-basic/tags.html change:
`{% for article in articles|sort %}`
to 
`{% for article in articles %}`

Also, I've tested both ways working in Python 2 but not Python 3. But I would like to confirm, did you just upgrade to using Python 3?

Thanks,
Josh

**Comment by salvor7** [comment]
I've been using Python 3 since we'll before I started using pelican.

**Comment by joshmorel** [comment]
Hi Andrew, I put in a PR - which should fix it #506

So was this theme working before for you or did you just start trying it? Maybe it was a combination of Python 3 and the latest Pelican breaking that piece of code? Either way, once the PR is merged it will work with both Python 2 and 3.

FYI - I had edited out  that part of my comment because I thought it was extraneous - but since you've replied to it I added it back in.
